### PR TITLE
Update Optimizer Help Doc

### PR DESCRIPTION
**Pull request by smk78**
## Description

This PR partly addresses a legacy problem highlighted by the issue below, namely that SasView currently uses a static version of optimizer.rst and the images therein. This had not been updated since the move to Qt or for a number of intervening Bumps releases.

This has been corrected here by taking a copy of the current Bumps optimizer.rst and copying it to SasView. The included images have also been replaced. Some minor text furtling has also been performed.

Fixes # (issue/issues)

https://github.com/SasView/sasview/issues/1808

## How Has This Been Tested?

Built locally.

## Review Checklist (please remove items if they don't apply):

- [x] Code has been reviewed
- [x] Functionality has been tested
- [ ] Windows installer (GH artifact) has been tested (installed and worked) 
- [ ] MacOSX installer (GH artifact) has been tested (installed and worked) 
- [x] User documentation is available and complete (if required)
- [ ] Developers documentation is available and complete (if required)
- [x] The introduced changes comply with SasView license (BSD 3-Clause)

### CODE REVIEW

**Comment by smk78** [comment]
**Note** @pkienzle / @bmaranville may well wish to port these changes back to Bumps.

**Review by lucas-wilkins — COMMENTED** [review]
The fit algorithm windows are currently being changed (in open PR's I think)

File: src/sas/qtgui/Perspectives/Fitting/media/optimizer.rst [hunk 1]
```diff
@@ -39,7 +39,7 @@ preliminary analysis, but only works reliably for weakly correlated parameters.
 For full uncertainty analysis, :ref:`fit-dream` uses a random walk to explore
 the parameter space near the minimum, showing pair-wise correlations
 amongst the parameter values.  In order for :ref:`fit-dream` to return the
-correct uncertainty, the function to be optimized should be a conditional
+correct uncertainy, the function to be optimized should be a conditional
```
Comment (lucas-wilkins) [inline src/sas/qtgui/Perspectives/Fitting/media/optimizer.rst:42]: Think it was spelled correctly before

File: src/sas/qtgui/Perspectives/Fitting/media/optimizer.rst [hunk 2]
```diff
@@ -121,15 +121,21 @@ otherwise it will restart the fit from a random point in the parameter space.
...
+``--fit=scipy.leastsq`` to restore the previous behaviour.
+
 References
 ----------
 
-.. [Levenberg1944]
+.. [#Levenberg1944]
     Levenberg, K.
-    *Quarterly Journal of Applied Mathematics*
+    *Quarterly Journal of Applied Mathmatics*
```
Comment (lucas-wilkins) [inline src/sas/qtgui/Perspectives/Fitting/media/optimizer.rst:135]: Nope. Also, the Journal is spelled with a lower case L.

File: src/sas/qtgui/Perspectives/Fitting/media/optimizer.rst [hunk 3]
```diff
@@ -347,12 +351,12 @@ within tolerance) and tiny (that is, all the points are close to each
 other) then the fit will terminate.  Use ``ftol=v`` and ``xtol=v`` from the
 command line.
 
-Use ``--fit=de`` to select differential evolution from the command line.
+Use ``--fit=de`` to select diffrential evolution from the command line.
```
Comment (lucas-wilkins) [inline src/sas/qtgui/Perspectives/Fitting/media/optimizer.rst:354]: No

**Review by lucas-wilkins — CHANGES_REQUESTED** [review]

File: src/sas/qtgui/Perspectives/Fitting/media/optimizer.rst (hunk 1 above)
Comment (smk78) [reply src/sas/qtgui/Perspectives/Fitting/media/optimizer.rst]: Good catch! Sorted.

File: src/sas/qtgui/Perspectives/Fitting/media/optimizer.rst (hunk 2 above)
Comment (smk78) [reply src/sas/qtgui/Perspectives/Fitting/media/optimizer.rst]: Another good catch! Also sorted.
(No L to correct)

File: src/sas/qtgui/Perspectives/Fitting/media/optimizer.rst (hunk 3 above)
Comment (smk78) [reply src/sas/qtgui/Perspectives/Fitting/media/optimizer.rst]: Yet another good catch! Also sorted.

**Review by lucas-wilkins — APPROVED** [review]

**Pull request merged**